src/transactions.js

```diff
@@ -81,6 +85,48 @@ export function unsignedMultisigTransaction(network, inputs, outputs) {
   return transactionBuilder.buildIncomplete();
 }
 
+/**
+ * Create an unsigned bitcoin transaction based on the network, inputs
+ * and outputs stored as a PSBT object
+ *
+ * Returns a [`PSBT`]{@link https://github.com/bitcoinjs/bitcoinjs-lib/blob/master/types/psbt.d.ts|PSBT} object from bitcoinjs-lib.
+ *
+ * @param {module:networks.NETWORKS} network - bitcoin network
+ * @param {module:inputs.MultisigTransactionInput[]} inputs - transaction inputs : NOTE - must be braid-aware
+ * @param {module:outputs.TransactionOutput[]} outputs - transaction outputs
+ * @returns {Psbt} an unsigned bitcoinjs-lib Psbt object
+ */
+export function unsignedMultisigPSBT(network, inputs, outputs) {
```

Comment:
I don't know the answer to this, so just asking, but would it make sense to just have this accept a multisig object? Don't all of these exist on the multisig or am I wrong to think that? Almost could be a method on a multisig object itself. `multisig.getUnsignedPSBT`

Reply:
No - it would have to support a potential array of multisigs, and the outputs are not always multisigs (only for change). So I'd say that the method signature should stay as is. Maybe if you wanted to generate a PSBT for a single UTXO in a slice, perhaps you might want to do that? Not sure